Packs/HPE_Service_Manager/Integrations/ServiceManager/Service_Manager.yml

```
@@ -163,6 +155,144 @@ script:
+            newRequest[params.resourcename]['AlertStatus'] = args.alertStatus;
+        }
+        if (args.area) {
+            newRequest[params.resourcename]['Area'] = args.area;
+        }
+        if (args.assignmentgroup) {
+            newRequest[params.resourcename]['AssignmentGroup'] = args.assignmentgroup;
+        }
+        if (args.affectedCI) {
+            newRequest[params.resourcename]['AffectedCI'] = args.affectedCI;
+        }
+        if (args.company) {
+            newRequest[params.resourcename]['Company'] = args.company;
+        }
+        if (args.category) {
+            newRequest[params.resourcename]['Category'] = args.category;
+        }
+        if (args.type) {
+            newRequest[params.resourcename]['Type'] = args.type;
+        }
+        if (args.phase) {
+            newRequest[params.resourcename]['Phase'] = args.phase;
+        }
+        if (args.status) {
+            newRequest[params.resourcename]['Status'] = args.status;
+        }
+        if (args.subarea) {
+            newRequest[params.resourcename]['Subarea'] = args.subarea;
+        }
+        if (args.contactmethod) {
+            newRequest[params.resourcename]['ContactMethod'] = args.contactmethod;
+        }
+        if (args.servicerecipient) {
+            newRequest[params.resourcename]['ServiceRecipient'] = args.servicerecipient;
+        }
+        if (args.affectedservice) {
+            newRequest[params.resourcename]['AffectedService'] = args.affectedservice;
+        }
+        if (args.servicecontact) {
+            newRequest[params.resourcename]['ServiceContact'] = args.servicecontact;
+        }
+        if (args.subcategory) {
+            newRequest[params.resourcename]['SubCategory'] = args.subcategory;
+        }
+        if (args.classification) {
+            newRequest[params.resourcename]['Classification'] = args.classification;
+        }
+        if (args.priority) {
+            newRequest[params.resourcename]['Priority'] = args.priority;
+        }
+        if (args.serviceprovidercompany) {
+            newRequest[params.resourcename]['ServiceProviderCompany'] = args.serviceprovidercompany;
+        }
+        if (args.updateactiontitle) {
+            newRequest[params.resourcename]['UpdateActionTitle'] = args.updateactiontitle;
+        }
+        if (args.serviceassociatedwith) {
+            newRequest[params.resourcename]['ServiceAssociatedWith'] = args.serviceassociatedwith;
+        }
+
+        //throw JSON.stringify(newRequest)
+        var res = doPost('/'+params.resourcename, newRequest);
+        var parsedRes = JSON.parse(res);
+        if (parsedRes.ReturnCode !== 0) {
+            throw 'Failed to create incident. Error Response: ' + res.Messages;
+        }
+
+        hrIncident = parsedRes[params.resourcename];
+        inc = parsedRes['Messages'][2];
+        incident = inc.split('"');
+
+        var srHPSM = {};
+        var lstHPSM = [];
+
+        srHPSM['ID'] = incident[1];
```

Why this message? Does it always exist and is built the same way?
Consider writing all 3 rows one by another to improve readability.
```suggestion
        var srHPSM = {};
        var lstHPSM = [];
        
        inc = parsedRes['Messages'][2];
        incident = inc.split('"');
        srHPSM['ID'] = incident[1];
```